libc/docs/gpu/building.rst

```diff
@@ -86,16 +86,13 @@ compiler. These tools must all be up-to-date with the libc source.
      -DCMAKE_C_COMPILER=$HOST_C_COMPILER      \
```

Comment:
@jhuber6 re-reading https://libc.llvm.org/gpu/building.html, I think we can/should remove `libc` from the `-DLLVM_ENABLE_PROJECTS` list on L85 now that building libc-hdrgen is no longer necessary. WDYT?

Reply:
Definitely, I was actually going to bring that up but forgot.

Reply:
4a474332c8f3